apps/btc_family/btc_xpub.c

```diff
@@ -302,10 +303,15 @@ void btc_get_xpub(btc_query_t *query) {
                                        init_req->derivation_paths_count);
   memzero(seed, sizeof(seed));
   if (true == status) {
-    send_xpubs(query, xpub_list, init_req->derivation_paths_count);
+    status = send_xpubs(query, xpub_list, init_req->derivation_paths_count);
   } else {
     // send unknown error; do not know failure reason
     btc_send_error(ERROR_COMMON_ERROR_UNKNOWN_ERROR_TAG, 1);
   }
+
+  if (!status) {
+    return;
+  }
+
   delay_scr_init(ui_text_check_cysync_app, DELAY_TIME);
```

Comment:
Nice catch! Just a nitpick, this can be changed to this to minimize obfuscation
```suggestion
  if (status) {
    delay_scr_init(ui_text_check_cysync_app, DELAY_TIME);
  }
```

Reply:
thanks, I have updated